package.json

```
@@ -3,9 +3,13 @@
   "version": "4.0.7",
   "description": "EventEmitter3 focuses on performance while maintaining a Node.js AND browser compatible interface.",
   "main": "index.js",
+  "exports": {
+    "import": "./index.mjs",
+    "require": "./index.js"
+  },
```

```suggestion
  "exports": {
    ".": {
      "import": "./index.mjs",
      "require": "./index.js"
    },
    "./package.json": "./package.json"
  },
```

This is another breaking change as people can no longer do something like this

```js
const EventEmitter = require('eventemitter3/umd/eventemitter3.js');
```

---

Sure, can change it to this:

```
  "exports": {
    ".": {
      "import": "./index.mjs",
      "require": "./index.js"
    },
    "./umd": "./umd/eventemitter3.js",
    "./package.json": "./package.json"
  },
```

Probabaly would make sense to add some tests on that.

---

I think there is no clean way. Anyway I think it is not needed.